sktime/registry/_base_classes.py

```diff
@@ -81,6 +81,8 @@
     _SeriesToSeriesTransformer,
 )
 
+from ..networks.base import BaseDeepNetwork
```

Comment:
please use absolute imports

Reply:
Ah, this seems to be a mistake by vsocde autocomplete. Will resolve this